scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
```

Comment:
:stop_sign: **Security Vulnerability**: This file contains hardcoded credentials and several security vulnerabilities. If this is for testing, it should be clearly marked and isolated[^1][^2][^3][^4].
```suggestion
// SECURITY: This section contains intentional vulnerabilities for testing purposes.
// DO NOT use this code in production environments.

const apiKey = process.env.API_KEY || "dummy_key" // Use environment variables
const password = process.env.PASSWORD || "dummy_password"
const dbConnection = process.env.DB_CONNECTION || "dummy_connection"

// Command injection vulnerability (for testing only)
function executeCommand(userInput: string) {
    // TODO: Implement safe command execution
}

// Path traversal vulnerability (for testing only)
function readUserFile(filename: string) {
    // TODO: Implement safe file reading with path validation
}

// SQL injection pattern (for testing only)
const query = `SELECT * FROM users WHERE id = ?` // Use parameterized queries
// TODO: Implement safe SQL query execution
```